packages/mjml-head-attributes/README.md

```diff
@@ -1,17 +1,22 @@
 ## mj-attributes
 
-This tag allows you to modify default MJML attributes on a `mj-tag` or on all tag with `mj-all`.
+Inside `mj-attributes`, a tag citing one MJML component (like `mj-text`;
+see example) overrides default settings for listed MJML attributes
+on the one component.
 
-In addition of all available `mj-tag`, you can also reuse a set of MJML attributes with `mj-class`.
+An `mj-all` is like the above, but affects every MJML component with one tag.
+
+`mj-class` tags create a named group of MJML attributes you can apply to MJML
+components much like you can apply HTML classes to HTML elements. To apply them, use `mj-class="<name>"`.
```

Comment:
I feel that we shouldn't mention similarity with CSS classes as it could be confused as css-class.

We may also add this feature that wasn't documented https://mjml.io/try-it-live/_oZvDHudAc9 `mj-class` can have children and will allow to update default attributes locally